atomicwrites/__init__.py

```diff
@@ -182,6 +182,13 @@ def get_fileobject(self, suffix="", prefix=tempfile.gettempprefix(),
             dir = os.path.normpath(os.path.dirname(self._path))
         descriptor, name = tempfile.mkstemp(suffix=suffix, prefix=prefix,
                                             dir=dir)
+
+        # If specified, change user and group.
+        uid = kwargs.pop('uid', -1)
+        gid = kwargs.pop('gid', -1)
+        if uid != -1 or gid != -1:
+            os.fchown(descriptor, uid, gid)
```

Comment:
Do we need check for whether `sys.platform` is on Unix?

https://docs.python.org/3/library/os.html#os.fchown

Reply:
Maybe an `assert` around it

Reply:
May need to ensure that if one is specified and the other isn't, we don't attempt to set either UID or GID to -1.

Reply:
@mookerji yes I was concerned about this as well. Do you know of a way to test for "unix"? The method that comes to mind is doing something like `hasattr(os, 'fchown')`

Reply:
@borgel, According to the docs `-1` will leave that field unchanged, so it is always a safe default. The if clause here was only added so we don't run the system call unnecessarily.